packages/orbit-components/src/primitives/TooltipPrimitive/__tests__/index.test.js

```diff
@@ -2,22 +2,20 @@
 import * as React from "react";
 import { render } from "@testing-library/react";
 
-// TODO: it is failing without it, seems like @testing-library does not see it in global config
-import "jest-styled-components";
 import Tooltip from "../index";
 import Airplane from "../../../icons/Airplane";
 
-describe("Tooltip", () => {
-  const content = "Write some message to the user";
-  jest.spyOn(React, "useMemo").mockImplementationOnce(() => "TooltipID50000-id-50000");
-
-  const { container } = render(
-    <Tooltip content={content}>
-      <Airplane />
-    </Tooltip>,
-  );
+jest.mock("../../../utils/randomID", () => name => `${name}50000-id-50000`);
 
+describe("Tooltip", () => {
   it("it should match snapshot", () => {
+    const content = "Write some message to the user";
+    const { container } = render(
+      <Tooltip content={content}>
+        <Airplane />
+      </Tooltip>,
+    );
+
     expect(container.firstChild).toMatchSnapshot();
```

Comment:
can we get rid of snapshot testing here? or it will be a part of another PR?

Reply:
I wanted to get rid of it at first, too, but there's quite a bit of functionality to cover 😕 so I think it's best to do it outside of this epic.